### ISSUE TITLE
"Illegal Instruction" error

### ISSUE THREAD

**Issue by ianglennon** [rm-hull/luma.led_matrix#269]
#### Type of Raspberry Pi
PiZero W

#### Linux Kernel version

Linux raspberrypi 5.10.63+ #1459 Wed Oct 6 16:40:27 BST 2021 armv6l GNU/Linux

#### Expected behaviour

I would expect python3 to not terminate when I run the command

from luma.led_matrix.device import max7219

#### Actual behaviour

```
pi@raspberrypi:~/luma.led_matrix/examples $ python3
Python 3.9.2 (default, Mar 12 2021, 04:06:34)
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from luma.led_matrix.device import max7219
Illegal instruction
pi@raspberrypi:~/luma.led_matrix/examples $ echo $?
132
pi@raspberrypi:~/luma.led_matrix/examples $
```

Output of pip3 list

```
Package         Version
--------------- ---------
cbor2           5.4.2
certifi         2020.6.20
chardet         4.0.0
colorzero       1.1
Deprecated      1.2.13
distro          1.5.0
gpiozero        1.6.2
idna            2.10
luma.core       2.3.1
luma.led-matrix 1.5.0
Pillow          9.0.0
pip             20.3.4
pyftdi          0.53.3
pyserial        3.5
python-apt      2.2.1
pyusb           1.2.1
requests        2.25.1
RPi.GPIO        0.7.0
setuptools      52.0.0
six             1.16.0
smbus2          0.4.1
spidev          3.5
ssh-import-id   5.10
urllib3         1.26.5
wheel           0.34.2
wrapt           1.13.3
```

When running ```from PIL import Image``` I get the same error

**Comment by thijstriemstra** [comment]
> When running from PIL import Image I get the same error

Can you try installing an older version of Pillow, e.g. `pip install "Pillow==8.4.0"`?

**Comment by ianglennon** [comment]
I'm going to hold this until #271 gets resolved.  If I can't install and run it on a 3B+ then I pretty much have no hope of getting it going on the ZeroW

**Comment by thijstriemstra** [comment]
Let me ask again; Can you try installing an older version of Pillow, e.g. pip install "Pillow==8.4.0"?

**Comment by ianglennon** [comment]
THIS IS NOT THE SAME ERROR!  IT IS NOT EVEN THE SAME RPI!

I'm done.

**Comment by thijstriemstra** [comment]
hey, im trying to help you.

**Comment by ianglennon** [comment]
Yeah, I know, sorry for the unnecessary all-caps response.  I have quite literally been trying to get this going all day and have had zero luck.  I have tried everything that has been suggested on the other tickets, all resulting in some kind of failure.  I have simply lost patience with it.  This was supposed to be the final and easiest step in designing and building my own device, starting off with a simple 7-digit display with a MAX7219 driver.  I had my own PCB fabricated, I spent the morning soldering components to it, and I even got to see it light up.  After that, nothing else has gone right.

I'm not bothered about the Pi Zero not working, I only used it because my 3B+ was busy doing something else.  Now that it's available I want to focus on that instead.  That's the other ticket, btw... #270 

I'll re-open that one (I honestly didn't realise I could re-open a ticket) and we'll try again.
